src/SFA.DAS.EmployerAccounts.Api.Types/AccountNameSummary.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.EmployerAccounts.Api.Types
+{
```

Comment:
file scoped namespace

Reply:
That library is using .NetStandard2.1 - which is C#8.0 - what do you want to do? I don't recommend forcing latest language features as that is a NuGet